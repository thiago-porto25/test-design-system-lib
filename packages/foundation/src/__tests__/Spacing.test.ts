import { Spacing } from '..';

test('Spacing must be readonly', () => {
  expect(() => {
    Spacing.xs = 'xs';
  }).toThrow();
});

test('snapshot of Spacing', () => {
  expect(Spacing).toMatchSnapshot();
});
