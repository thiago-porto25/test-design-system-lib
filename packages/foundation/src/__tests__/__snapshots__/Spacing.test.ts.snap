// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`snapshot of Spacing 1`] = `
Object {
  "lg": "lg",
  "md": "md",
  "sm": "sm",
  "xl": "xl",
  "xs": "xs",
  "xxl": "xxl",
  "xxs": "xxs",
  "xxxl": "xxxl",
  "xxxs": "xxxs",
}
`;
